npm/react/examples/find-webpack/cypress/plugins/index.js

```diff
@@ -10,9 +15,9 @@ module.exports = (on, config) => {
   const webpackConfig = findReactScriptsWebpackConfig(config)
 
   const rules = webpackConfig.module.rules.find((rule) => !!rule.oneOf).oneOf
-  const babelRule = rules.find((rule) => /babel-loader/.test(rule.loader))
+  const babelRule = rules.find((rule) => typeof rule.loader === 'string' && /babel-loader/.test(rule.loader))
 
-  babelRule.options.plugins.push(require.resolve('babel-plugin-istanbul'))
+  typeof babelRule.options !== 'string' && babelRule.options.plugins.push(require.resolve('babel-plugin-istanbul'))
```

Comment:
Are these changes related to the `on`/`config` -> `options` signature change, or something else?

Reply:
To test my new types, I used `// @ts-check`.
This makes the ts-check pass.